k8s/base/aws_eks/deployments/mpm-backend.yaml

```diff
@@ -23,3 +23,17 @@ spec:
                 name: mpm-configmap-backend
             - secretRef:
                 name: mpm-secret-backend
+          volumeMounts:
+            - name: storage-data
+              mountPath: /var/www/html/storage
+              subPath: ""
+            - name: logs-data
+              mountPath: /var/www/html/storage/logs
+              subPath: logs
```

Comment:
I don't think we need a separate `logs` mount here. After all, we are not even using logging to file anymore. It was moved to STDERR/STDOUT logging which works much better with `k8s`.

(applies to all files in this PR)

Reply:
Noted.